pkg/common/config/config_yaml_test.go

```diff
@@ -65,6 +65,17 @@ vcenter:
     secretNamespace: kube-system
 `
 
+const badConfigYAML = `
```

Comment:
Please add more test cases to cover different inputs

- valid yaml
- valid ini
- invalid yaml
- invalid ini
- invalid format

Reply:
ok, I'll add more tests.